csvs/card.csv

```diff
@@ -1523,9 +1523,11 @@ CRU185	CRU	Promise of Plenty	3	0	1	2			R	Generic, Action, Attack	Go again				"If
 If Promise of Plenty is played from arsenal, it gains **go again.**"		Generic Action – Attack	Alexander Gering				No													S R – CRU185 – F, S R – CRU185 – U	https://storage.googleapis.com/fabmaster/cardfaces/2020-CRU/CRU185.png – CRU185 – F, https://storage.googleapis.com/fabmaster/cardfaces/2021-U-CRU/U-CRU185.png – CRU185 – U			
 IRA011, KSU028, CRU186	IRA, KSU, CRU	Lunging Press	3	0		2			P, C, R	Generic, Attack Reaction					Target attack action card gains +1{p}		Generic Attack Reaction	Sergey Gurskiy																	S – IRA011 – N, S – KSU028 – N, S R – CRU186 – F, S R – CRU186 – U	https://storage.googleapis.com/fabmaster/media/images/IRA011-P_6omATzN.width-450.png – IRA011 – N, https://storage.googleapis.com/fabmaster/media/images/KSU28.width-450.png – KSU028 – N, https://storage.googleapis.com/fabmaster/cardfaces/2020-CRU/CRU186.png – CRU186 – F, https://storage.googleapis.com/fabmaster/cardfaces/2021-U-CRU/U-CRU186.png – CRU186 – U			
 BVO015, IRA012, KSU021, RNR022, TEA019, 1HP374, CRU187	BVO, IRA, KSU, RNR, TEA, 1HP, CRU	Springboard Somersault	2	0		2			C, P, C, C, C, R, R	Generic, Defense Reaction					If Springboard Somersault is played from arsenal, it gains +2{d}.		Generic Defense Reaction	Alexander Mokhov																	S – BVO015 – N, S – IRA012 – N, S – KSU021 – N, S – RNR022 – N, S – TEA019 – N, S – 1HP374 – N, S R – CRU187 – F, S R – CRU187 – U	https://storage.googleapis.com/fabmaster/media/images/BVO15.width-450.png – BVO015 – N, https://storage.googleapis.com/fabmaster/media/images/IRA012-P_kmiIZwp.width-450.png – IRA012 – N, https://storage.googleapis.com/fabmaster/media/images/KSU21.width-450.png – KSU021 – N, https://storage.googleapis.com/fabmaster/media/images/RNR22.width-450.png – RNR022 – N, https://storage.googleapis.com/fabmaster/media/images/TEA19.width-450.png – TEA019 – N, https://storage.googleapis.com/fabmaster/media/images/1HP374.width-450.png – 1HP374 – N, https://storage.googleapis.com/fabmaster/cardfaces/2020-CRU/CRU187.png – CRU187 – F, https://storage.googleapis.com/fabmaster/cardfaces/2021-U-CRU/U-CRU187.png – CRU187 – U			
-CRU188	CRU	Cash In	2	4		2			R	Generic, Action					"You may destroy 4 Coppers, 2 Silvers, or 1 Gold you control rather than pay Cash In's {r} cost.
+CRU188	CRU	Cash In	2	4		2			R	Generic, Action	Go again				"You may destroy 4 Coppers, 2 Silvers, or 1 Gold you control rather than pay Cash In's {r} cost.
```

Comment:
There should be a comma between Action and Go Again here :D

Reply:
I could be wrong but I think those are two different fields - Generic, Action is from Types and Go again is from Card Keywords

Reply:
Agh, you're right >.> This is what I get for trying to review CSVs in browser 😂 Approved, thanks for the PR!